From e0474dc25e09d2f87a7f6b541016e873a0834854 Mon Sep 17 00:00:00 2001
From: Thomas White <tomwhite@apache.org>
Date: Tue, 18 Dec 2012 15:26:29 +0000
Subject: [PATCH 173/795] MR1: MAPREDUCE-4845. ClusterStatus.getMaxMemory() and getUsedMemory() exist in MR1 but not MR2. Contributed by Sandy Ryza.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1423472 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit ccee91c60f729843d4309142d24f8acf0b002eac)
(cherry picked from commit 132ff7182f2c6e15cc85bb7e31cec659c8088fa2)
(cherry picked from commit 9aee7a4bfb34353be2c7e85206b877ce2df5dc8d)
---
 .../org/apache/hadoop/mapred/ClusterStatus.java    |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/mapred/org/apache/hadoop/mapred/ClusterStatus.java b/src/mapred/org/apache/hadoop/mapred/ClusterStatus.java
index 979a847..e332bd2 100644
--- a/src/mapred/org/apache/hadoop/mapred/ClusterStatus.java
+++ b/src/mapred/org/apache/hadoop/mapred/ClusterStatus.java
@@ -296,6 +296,7 @@ public JobTrackerStatus getJobTrackerStatus() {
    * 
    * @return the size of heap memory used by the <code>JobTracker</code>
    */
+  @Deprecated
   public long getUsedMemory() {
     return used_memory;
   }
@@ -305,6 +306,7 @@ public long getUsedMemory() {
    * 
    * @return the configured size of max heap memory that can be used by the <code>JobTracker</code>
    */
+  @Deprecated
   public long getMaxMemory() {
     return max_memory;
   }
-- 
1.7.0.4

